### PR TITLE
added docker-tag command

### LINKED ISSUE #166: docker tag image command
`docker tag SOURCE_IMAGE:tag DESTINATION_IMAGE:tag`

### PR DESCRIPTION
**Pull request by thilakarajk**
Fixes #166
- Added docker tag command

### CODE REVIEW

File: commands/docker/docker-tag.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+### Provide new tag to docker image
+
+`docker tag {{EXISTING_IMAGE_NAME[:TAG]/ID}} {{NEW_IMAGE_NAME[:TAG]}}}`
```
Comment (arshadkazmi42) [inline commands/docker/docker-tag.md]: `{{EXISTING_IMAGE_NAME[:TAG]/ID}}` => `{{EXISTING_IMAGE_NAME[:TAG]}}`
Let's keep it like this and as in description below we already have mentioned either image name or id can be used

File: commands/docker/docker-tag.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+- <b>EXISTING_IMAGE_NAME[:TAG]/ID: </b> Name with tag or image id of the image
```
Comment (arshadkazmi42) [inline commands/docker/docker-tag.md]: `EXISTING_IMAGE_NAME[:TAG]/ID` => `EXISTING_IMAGE_NAME[:TAG]`

File: commands/docker/docker-tag.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+- <b>NEW_IMAGE_NAME[:TAG]: </b>New name for the existing image. If `[:TAG]` is not provided, by default `:latest` tag will be added.
+
+#### Example:
+
+`docker tag ubuntu ubuntu:new`
```
Comment (arshadkazmi42) [inline commands/docker/docker-tag.md]: `docker tag ubuntu ubuntu:new` => `docker tag ubuntu:latest ubuntu:new`

File: commands/docker/docker-tag.md (hunk 1 above)
Comment (thilakarajk) [reply commands/docker/docker-tag.md]: Sure. Will update that

File: commands/docker/docker-tag.md (hunk 2 above)
Comment (thilakarajk) [reply commands/docker/docker-tag.md]: Updated in the new commit

File: commands/docker/docker-tag.md (hunk 3 above)
Comment (thilakarajk) [reply commands/docker/docker-tag.md]: Updated in the new commit

File: commands/docker/docker-tag.md (hunk 1 above)
Comment (thilakarajk) [reply commands/docker/docker-tag.md]: Updated in the new commit

**Pull request merged**